packages/__examples__/.storybook/webpack.config.js

```diff
@@ -40,23 +39,20 @@ module.exports = ({ config, mode }) => {
       }
     }
   }
+  // Disable linting for Storybook builds, it fails the build when it tries to
+  // lint its autogenerated files.
+  baseConfig.module.rules = rules.filter(
+    (rule) => rule.loader !== 'eslint-loader'
+  )
```

Comment:
Our webpack config does linting, I think we should remove it, it leads to some weird scenarios. For example it tries to link autogenerated files by Storybook